conda/common/configuration.py

```diff
@@ -54,14 +54,9 @@
     from ruamel.yaml.reader import ReaderError
     from ruamel.yaml.scanner import ScannerError
 except ImportError:  # pragma: no cover
-    try:
-        from ruamel_yaml.comments import CommentedMap, CommentedSeq
-        from ruamel_yaml.reader import ReaderError
-        from ruamel_yaml.scanner import ScannerError
-    except ImportError:
-        raise ImportError(
-            "No yaml library available. To proceed, conda install ruamel.yaml"
-        )
+    raise ImportError(
+        "No yaml library available. To proceed, conda install ruamel.yaml"
```

Comment:
Should this message go away or be replaced with "reinstall conda with a different environment's conda"?

Reply:
Good question!